src/ui/profile_preferences.ui

```diff
@@ -198,7 +198,7 @@
              <bool>true</bool>
             </property>
             <property name="toolTip">
-             <string>&lt;p&gt;If you are playing a non-English game and seeing � instead of text, or special letters like &lt;span style=&quot; font-weight:600;&quot;&gt;ñ&lt;/span&gt; aren't showing right - try changing the encoding to UTF-8 or to one suggested by your game.&lt;/p&gt;</string>
+             <string>&lt;p&gt;If you are playing a non-English game and seeing � instead of text, or special letters like &lt;span style=&quot; font-weight:600;&quot;&gt;ñ&lt;/span&gt; aren't showing right - try changing the encoding to UTF-8 or to one suggested by your game.&lt;/p&gt;&lt;p&gt;For some encodings on some Operating Systems Mudlet itself has to provide the codec needed; if that is the case for this Mudlet then there will be a &lt;tt&gt;m &lt;/tt&gt; prefixed applied to those encoding names (so if they have errors the blame can be applied correctly!)&lt;/p&gt;</string>
```

Comment:
I'd like this tooltip to stay as-is because the new information does not really provide any useful information for the player, just gives more text to read which to a non-techie might be confusing.

Reply:
Then you will have people saying WTF does that "m" mean. Also it give the hint that if they are getting odd characters when they have a need for an encoder in that group - like the Medival people - they can appreciate that we ought to be able to fix it if they get in touch.

Reply:
That is the purpose of `getMudletInfo()` and the internal codec should not leak to the tooltip, if it is!